sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Integration.Tests/DistroWebAppLiveTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+using Azure.Core.TestFramework;
+using Azure.Monitor.Query;
+using Azure.Monitor.Query.Models;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using OpenTelemetry.Logs;
+using OpenTelemetry.Resources;
+
+#if NET6_0_OR_GREATER
+namespace Azure.Monitor.OpenTelemetry.AspNetCore.Integration.Tests
+{
+    public class DistroWebAppLiveTests : RecordedTestBase<MonitorExporterTestEnvironment>
+    {
+        private const string _testServerUrl = "http://localhost:9998/";
+
+        // DEVELOPER TIP: Change roleName to something unique when working locally (Example "Test##") to easily find your records.
+        // Can search for all records in the portal using this query:   Union * | where AppRoleName == 'Test##'
+        private const string _roleName = nameof(DistroWebAppLiveTests);
+
+        private const string _logMessage = "Message via ILogger";
+
+        private LogsQueryClient? _logsQueryClient = null;
+
+        // DEVELOPER TIP: Can pass RecordedTestMode.Live into the base ctor to run this test with a live resource.
+        // DEVELOPER TIP: Can pass RecordedTestMode.Record into the base ctor to re-record the SessionRecords.
+        public DistroWebAppLiveTests(bool isAsync) : base(isAsync) { }
+
+        [RecordedTest]
+        public async Task VerifyDistro()
+        {
+            // SETUP LOGS CLIENT
```

Comment:
Should we call logs or telemetry. Logs is also a signal, could make it hard to read the code.

Reply:
Good point! i'll make the change